src/views/Home/Pricing.vue

```diff
@@ -1,9 +1,16 @@
 <template>
   <Tile bgcolor="bg-wri-blue" title="Conference Pricing" titleColor="text-wri-teal">
     <div class="flex flex-wrap justify-center uppercase text-center">
-      <PricingBox name="Early Bird" price="99" link="javascript:void(0);"></PricingBox>
-      <PricingBox name="Regular" price="129" link="javascript:void(0);"></PricingBox>
-      <PricingBox name="Late Bird" price="139" link="javascript:void(0);"></PricingBox>
+      <PricingBox
+        v-if="showTicketSales"
+        name="Early Bird"
+        price="99"
+        link="https://www.tickettailor.com/events/werockitconf/268601"
+        button="Buy Now"
+      ></PricingBox>
+      <PricingBox v-else name="Regular" price="99" link="javascript:void(0);" button="Available Soon"></PricingBox>
+      <PricingBox name="Regular" price="129" link="javascript:void(0);" button="Available Soon"></PricingBox>
```

Comment:
For the "Available Soon" for the Regular and Late Birds.  Should we put the dates that those go live?  @x86octopus @RBangel

Reply:
I will create additional stories to cover this in Pivitol Tracker. :) Everything looks amazing!